graph-manager-docs/source/operation-registry.mdx

```diff
@@ -213,9 +213,9 @@ const gateway = new ApolloGateway({
 
 </ExpansionPanel>
 
-#### 5.1 Optionally, set the schema tag
+#### 5.1 Optionally, set the graph variant (AKA schemaTag)
```

Comment:
leaving this reference since the Apollo Server / plugin option is still `schemaTag`